[dsmr] Update line endings

wborn opened the pull request:
In #8383 a .gitattributes file was modified but not all files were updated for the change.
On Linux I now often have modified files on new clones or when switching between branches.

Hilbrand commented:
What did you change that causes the diffs? the line endings? or encoding? I'm curious why this wasn't correct, as if it wasn't correct I would have expected the unit tests to fail.

wborn commented:
I haven't changed anything myself but Git does:

```
$ git clone git@github.com:openhab/openhab-addons.git
Cloning into 'openhab-addons'...
remote: Enumerating objects: 35, done.
remote: Counting objects: 100% (35/35), done.
remote: Compressing objects: 100% (20/20), done.
remote: Total 371285 (delta 6), reused 27 (delta 4), pack-reused 371250
Receiving objects: 100% (371285/371285), 370.27 MiB | 11.11 MiB/s, done.
Resolving deltas: 100% (191921/191921), done.
$ cd openhab-addons/
$ git status
On branch 2.5.x
Your branch is up to date with 'origin/2.5.x'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   bundles/org.openhab.binding.dsmr/src/test/resources/org/openhab/binding/dsmr/internal/Iskra_AM550.telegram
	modified:   bundles/org.openhab.binding.dsmr/src/test/resources/org/openhab/binding/dsmr/internal/Landis_Gyr_E350.telegram
	modified:   bundles/org.openhab.binding.dsmr/src/test/resources/org/openhab/binding/dsmr/internal/Landis_Gyr_ZCF110.telegram
	modified:   bundles/org.openhab.binding.dsmr/src/test/resources/org/openhab/binding/dsmr/internal/Sagemcom_XS210.telegram
	modified:   bundles/org.openhab.binding.dsmr/src/test/resources/org/openhab/binding/dsmr/internal/ace4000.telegram
	modified:   bundles/org.openhab.binding.dsmr/src/test/resources/org/openhab/binding/dsmr/internal/dsmr_40.telegram
	modified:   bundles/org.openhab.binding.dsmr/src/test/resources/org/openhab/binding/dsmr/internal/dsmr_42.telegram
	modified:   bundles/org.openhab.binding.dsmr/src/test/resources/org/openhab/binding/dsmr/internal/dsmr_50.telegram
	modified:   bundles/org.openhab.binding.dsmr/src/test/resources/org/openhab/binding/dsmr/internal/flu5.telegram
	modified:   bundles/org.openhab.binding.dsmr/src/test/resources/org/openhab/binding/dsmr/internal/smarty.telegram
	modified:   bundles/org.openhab.binding.dsmr/src/test/resources/org/openhab/binding/dsmr/internal/unregistered_meter.telegram

no changes added to commit (use "git add" and/or "git commit -a")
```

According to my editor these files are UTF-8 encoded with Windows line endings.

Hilbrand reviewed (approved):
LGTM
